labels/en-US.json

```diff
@@ -127,6 +127,7 @@
   "createAccount": "Create an account for easy checkout on future orders",
   "createdBy": "Created By",
   "createdDate": "Created Date",
+  "createQuote": "Create Quote",
```

Comment:
Same as above. Please place this in the right spot alphabetically. Use an alphasort to check.

Reply:
Added at correct location